ts/features/fastLogin/store/reducers/optInReducer.ts

```diff
@@ -0,0 +1,49 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { PersistConfig, persistReducer } from "redux-persist";
+import { getType } from "typesafe-actions";
+import { setFastLoginOptIn } from "../actions/optInActions";
+import { Action } from "../../../../store/actions/types";
+import {
+  logoutFailure,
+  logoutSuccess
+} from "../../../../store/actions/authentication";
+
+export type FastLoginOptInState = {
+  fastLoginEnabled: boolean | undefined;
```

Comment:
Do we need to repeat "fastLogin"? We are already in the section of the store regarding fast login isn't "enabled" enough?